keps/sig-node/20180906-quotas-for-ephemeral-storage.md

```diff
@@ -622,15 +625,21 @@ appropriate end to end tests.
 The following criteria applies to
 `LocalStorageCapacityIsolationFSMonitoring`:
 
-Alpha
+## Phase 1: Alpha (1.15)
 
 - Support integrated in kubelet
+- Alpha-level documentation
+- Feature gate disabled by default
+- Unit test coverage
+- Node e2e test
 
-Beta
+## Phase 2: Beta (target 1.16)
```

Comment:
Since some of the objectives for the feature are a reduction in metric latency and overhead from metrics collection.  I think we need benchmarks or other validation that the feature accomplishes those goals to move to beta.

Reply:
I'm trying to work up a benchmark/stress test for this.  The basic method I'm trying to use is many pods using emptydirs with many directories and files within them.

Reply:
Sounds good.  The statement here doesn't have to be specific at all.  Something to the effect of "benchmark to determine latency, resource cost, and syscall overhead (?) of using quotas relative to existing monitoring solution"

Reply:
OK, done.

Reply:
Added microbenchmark results.